mylist.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*initial)
```

Comment:
- [ ] MyList class should have an instance variable `@list`, kindly consider implementing this inside the `initialize` method so the `MyList` class has a list